src/js/components/orgs/taskOrgCard.tsx

```diff
@@ -175,12 +177,53 @@ const TaskOrgCard = ({
     type === ORG_TYPES.QA && org && orgCommitIdx !== 0,
   );
 
+  let popover = null;
+
+  switch (type) {
+    case ORG_TYPES.QA:
+      popover = (
+        <TourPopover
+          align="top left"
+          heading={i18n.t('Tester & Test Org')}
+          body={
+            <Trans i18nKey="tourTaskTestOrg">
+              Assign yourself or someone else as Tester on this Task. Testers
+              create a Test Org to view the Developer’s work, and then approve
+              the work or request changes that must be addressed before the Task
+              can be completed. Use the drop down menu to change or remove the
+              Tester.
+            </Trans>
+          }
+        />
+      );
+      break;
+    case ORG_TYPES.DEV:
+      popover = (
+        <TourPopover
+          align="top left"
+          heading={i18n.t('Developer & Dev Org')}
+          body={
+            <Trans i18nKey="tourTaskDevOrg">
+              A Developer is the person assigned to do the work of a Task.
+              Developers create Dev Orgs for their work, retrieve their changes,
+              and then submit their work for someone to test. Assign yourself or
+              another Collaborator as the Developer on this Task. Use the drop
+              down menu to change or remove the Developer.
+            </Trans>
+          }
+        />
+      );
+      break;
+  }
+
   return (
     <div
       className="slds-size_1-of-1
         slds-large-size_1-of-2
-        slds-p-around_x-small"
+        slds-p-around_x-small
+        slds-is-relative right-edge"
```

Comment:
@dvdherron if we aren't using right-edge anywhere (currently on the task and epic status sidebars I think) then we could use something on the task cards to push them a little closer to their corner's edge for consistency? Although, we may want a different classname than `right-edge` since that doesn't seem to indicate what it is doing.

Reply:
made an `.org-card` class and moved them closer

![image](https://user-images.githubusercontent.com/41588129/119975666-6c7cbf80-bfb6-11eb-8242-338f715329a6.png)